engine/objconv/basemaker/base_maker.cpp

```diff
@@ -80,252 +80,6 @@ int nb_checks = 1;
 double last_check = 1;
 double cur_check = 1;
 
-/*
```

Comment:
This may have been a mistake to delete code from this particular file. Will need to revisit.

Reply:
BaseMaker itself is something I haven't used at all. AFAICT, it's a way to run base code without running the game. 

This specific code inside was all commented out, including a second copyright notice.

If you still feel strongly about it, I suggest you either uncomment it and figure out if it's critical (probably not) or open an issue. 

There are a whole bunch of places where the original devs commented out stuff for "5 minutes". You have to be part archeologist, part seer to figure out what and why.